crates/anvil/tests/it/pubsub.rs

```diff
@@ -246,8 +246,6 @@ async fn test_subscriptions() {
     assert_eq!(blocks, vec![1, 2, 3])
 }
 
-// TODO: Fix this, num > 17 breaks the test due to poller channel_size defaults to 16. Recv channel
-// lags behind.
```

Comment:
I assume this has been fixed already since the test passes with no `ignore` and `n = 1000`

Reply:
yep